source/helpers/bisect.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+
+import features from '../features';
+
+export default async function bisectFeatures(): Promise<Record<string, boolean>> {
+	const bisectedFeatures: string[] | false | undefined = await cache.get('bisect');
+	if (bisectedFeatures === undefined) {
+		return {};
+	}
+
+	if (bisectedFeatures === false) {
+		createMessageBox('Every feature has been disabled. If you still see the bug, try disabling the whole extension.', false);
+		return {};
+	}
+
+	const firstHalf = bisectedFeatures.slice(0, Math.ceil(bisectedFeatures.length / 2));
+	if (firstHalf.length === 0) {
+		createMessageBox('Every feature has been disabled. Can you see the bug?');
+	} else if (firstHalf.length === 1) {
+		createMessageBox(<span>The bug is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + firstHalf[0] + '.tsx'}><code>{firstHalf[0]}</code></a>.</span>, false);
+	} else {
+		createMessageBox(`${Math.ceil(Math.log2(firstHalf.length))} steps remaining. Can you see the bug?`);
+	}
+
+	return Object.fromEntries(features.list.map(feature => [`feature:${feature}`, firstHalf.includes(feature)]));
+}
+
+function createMessageBox(message: string | Element, yesNoButtons = true): void {
+	document.body.append(
+		<div className="Box" style={{position: 'fixed', top: 0, left: 0}}>
```

Comment:
```suggestion
		<div className="Box p-3" style={{position: 'fixed', bottom: 10, left: '50%', transform: 'translateX(-50%)'}}>
```

<img width="567" alt="Screen Shot 2021-04-04 at 00 15 27" src="https://user-images.githubusercontent.com/1402241/113486023-eb83d900-94da-11eb-9609-cc88d285c5e2.png">

Probably also the width needs to be limited somehow. Alternative the horizontal centering could be achieved via flexbox + a `pointer-events:none` full-width container